### ISSUE TITLE
Function Scope Configuration: unrecognized property 'lumigo'

### ISSUE THREAD

**Issue by Nxtra** [lumigo-io/serverless-lumigo-plugin#84]
I recently updated my serverless framework version from `2.10` to `2.68`.

Now it gives me the following configuration warning on the function scope configuration property of the lumigo plugin: 
_Serverless:   at 'functions.myFunctionName': unrecognized property 'lumigo'_
For a function defined like this:
```
functions:
  myFunctionName:
    handler: functions/my-function.handler
    lumigo:
      enabled: false
```

Should I be worried about this? 
If so, how can I resolve it?

**Comment by nirLumigo** [comment]
Hi, thanks for opening the issue.
We will check it.

**Comment by nirLumigo** [comment]
Hi @Nxtra , we investigated the issue and we found that you shouldn't be worried, this warning doesn't affect you.
We will release a new version without the warning.

**Comment by nirLumigo** [comment]
Hi @Nxtra , we released a fix on v1.13.0.

**Comment by Nxtra** [comment]
@nirLumigo, great!
The warning is gone.
